portal/static/js/main.js

```diff
@@ -1669,24 +1680,24 @@ OrgTool.prototype.getDefaultModal = function(o) {
                 '<div class="modal-content">' +
                 '<div class="modal-header">' +
                 '<button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>' +
-                '<h4 class="modal-title">Consent to share information</h4>' +
+                '<h4 class="modal-title">' + i18next.t("Consent to share information") + '</h4>' +
                 '</div>' +
                 '<div class="modal-body">' +
                 '<h4>Terms</h4>' +
-                '<p>I consent to sharing information with the <span class="consent-clinic-name">' + orgName + '.</span></p>' +
+                '<p>' + i18next.t("I consent to sharing information with the ") + '<span class="consent-clinic-name">' + orgName + '.</span></p>' +
```

Comment:
Can you try i18next interpolation here? Some languages may have the org name in the beginning of the sentence.